beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -723,7 +723,27 @@ func (s *Server) PublishBlockV2(w http.ResponseWriter, r *http.Request) {
 		s.publishBlock(ctx, w, r, true)
 	}
 }
+func (s *Server) versionHeaderFromCurrentSlot() string {
```

Comment:
Rename to `versionHeaderFromCurrentFork` in case you decide to use `CurrentFork()`